WebRoot/js/zimbraAdmin/servers/view/ZaNewVolumeXWizard.js

```diff
@@ -0,0 +1,1062 @@
+
+    this.stepChoices.push({
+        value: ZaNewVolumeXWizard.NEW_CEPH_BUCKET,
+        label: ZaMsg.TABT_CephBucketPage,
+    });
+    cases.push({
+        type: _CASE_,
+        tabGroupKey: ZaNewVolumeXWizard.NEW_CEPH_BUCKET,
+        caseKey: ZaNewVolumeXWizard.NEW_CEPH_BUCKET,
+        numCols: 1,
+        items: [
+            {
+                type: _ZAWIZGROUP_,
+                colSizes: ["200px", "*"],
+                numCols: 2,
+                items: [
+                    {
+                        ref: "bucketName",
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_BucketName,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        ref: "accessKey",
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_AccessKey,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        ref: "secretKey",
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_SecretAccessKey,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        ref: "destinationPath",
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_DestinationPath,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        ref: "url",
+                        type: _TEXTFIELD_,
+                        label: ZaMsg.LBL_VM_URL,
+                        labelLocation: _LEFT_,
+                        labelCssStyle: "text-align:left;",
+                        width: 150,
+                    },
+                    {
+                        type: _GROUP_,
+                        colSizes: ["200px", "*"],
+                        colSpan: 2,
+                        items: [
+                            { type: _CELLSPACER_ },
+                            {
+                                type: _BUTTON_,
+                                label: ZaMsg.LBL_VM_TestBucket,
+                            },
+                        ],
+                    },
+                ],
+            },
+        ],
+    });
+
+    // Left sidebar visibility checks
+    var labelVisibility = {};
+    labelVisibility[ZaNewVolumeXWizard.GENERAL_STEP] = {};
+    labelVisibility[ZaNewVolumeXWizard.NEW_INTERNAL_VOLUME] = {
+        checks: [[ZaNewVolumeXWizard.isStep, "InternalVolume", true]],
```

Comment:
done

Reply:
`ZaNewVolumeXWizard.NEW_INTERNAL_VOLUME` is set as follows:
```
ZaNewVolumeXWizard.NEW_INTERNAL_VOLUME = ++this.TAB_INDEX;
```
Does the change work in the same way as before?
```
(before)
checks: [[ZaNewVolumeXWizard.isStep, "InternalVolume", true]], 

(after)
checks: [[ZaNewVolumeXWizard.isStep, ZaNewVolumeXWizard.NEW_INTERNAL_VOLUME, true]],
```

Reply:
yes, the functionality is the same since that constant represents an integer, I just didn't want to define another one because it's only used in those checks

Reply:
I got it. Thanks.